content/docs/tasks/security/mtls-migration/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Mutual TLS Migration
+description: Shows you how to migrate your Istio services to mutual TLS incrementally.
+weight: 10
+keywords: [security,authentication,migration]
+---
+
+This task shows how to migrate your existing Istio services' traffic from plain text to mutual TLS without breaking live traffic.
+
+In practice, a mesh consists of both Istio services (with Envoy sidecar) and services without Envoy sidecar(call it "legacy" below for simplicity).
+A legacy service can't use Istio issued key/certificate to send mutual TLS traffic. We want to enable mutual TLS incrementally, safely.
+
+## Before you begin
+
+* Understand Istio [authentication policy](/docs/concepts/security/authn-policy/) and related [mutual TLS authentication](/docs/concepts/security/mutual-tls/) concepts.
+
+* Have a Kubernetes cluster with Istio installed, without global mutual TLS enabled (e.g use `install/kubernetes/istio-demo.yaml` as described in [installation steps](/docs/setup/kubernetes/quick-start/#installation-steps), or set `global.mtls.enabled` to false using [Helm](/docs/setup/kubernetes/helm-install/)).
+
+* For demo, create three namespaces `foo`, `bar`, `legacy`, and deploy [httpbin](https://github.com/istio/istio/blob/{{<branch_name>}}/samples/httpbin) and [sleep](https://github.com/istio/istio/tree/master/samples/sleep) with sidecar on both of them. Also, run another sleep app without sidecar (to keep it separate, run it in `legacy` namespace)
```

Comment:
Replace https://github.com uses with {{< github_tree >}}. See other .md files in the repo for examples.

Reply:
done